### PR TITLE
RK-10245 - live logger first draft

### PR DESCRIPTION
**Pull request by Rookeret73**
First draft of the Live Logger documentation update.
Do not merge yet - the video link is pending an update and upload to the youtube channel.

### CODE REVIEW

File: website/yarn.lock [hunk 1]
```diff
@@ -2,826 +2,774 @@
 # yarn lockfile v1
```
Comment (niryarden) [inline website/yarn.lock:3]: i guess you did not try to edit this file, right?

File: docs/live-logger.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Free text Filtering
+
+Use the intuitive and robust Free Text Search to only switch on logs that contain a specific string.
+This will help you pinpoint your search to logs related to a specific error message, function or returned value.
+
+## Context based Filtering
+
+Rookout integrates with OpenTracing based tools, as well as with MDC.
+Use the Filter By field only switch on logs that match the wanted context.
```
Comment (niryarden) [inline docs/live-logger.md:20]: something is off in this sentence.... isn't it?

File: docs/live-logger.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (niryarden) [inline docs/live-logger.md:1]: maybe something in this page about connecting to a third party target?

File: docs/live-logger.md (hunk 2 above)
Comment (Rookeret73) [reply docs/live-logger.md:20]: yup, missing a 'to'. thanks!

File: docs/live-logger.md (hunk 3 above)
Comment (Rookeret73) [reply docs/live-logger.md:1]: Thinking about it. Targets are not unique to Live Logger, as are SDK install, server selection, etc.
But it is a core part of the Live Logger story (maybe more so than in the Live Debugger story).
So maybe I'll add it anyway.

File: website/yarn.lock (hunk 1 above)
Comment (Rookeret73) [reply website/yarn.lock:3]: definitely not. I'll unstage it. thanks.

File: docs/node-setup.md [hunk 4]
```diff
@@ -54,6 +54,7 @@ The `start` method is used to initialize the SDK. Receives configuration using a
 | `debug` | `ROOKOUT_DEBUG` | False | Set to `True` to increase log level to debug |
 | `throw_errors` | None | False | Set to `True` to reject the promsise or throw an exception if `start` fails (error message will not be printed in console) |
 | `sources` | `ROOKOUT_SOURCES` | None | Source information (see below) |
+| NONE | `ROOKOUT_LIVE_LOGGER` | False | Set to `True` to enable Rookout Live Logger |
```
Comment (niryarden) [inline docs/node-setup.md:57]: in my winston2 PR i added `live_logger` as an equivalent param of rookout.start().
so you can add it

**Review by lirook — APPROVED** [review]

**Pull request merged**